Libraries/src/Amazon.Lambda.AspNetCoreServer/AbstractAspNetCoreFunction.cs

```diff
@@ -259,6 +303,40 @@ private protected bool IsStarted
             }
         }
 
+        private void AddRegisterBeforeSnapshot()
+        {
+            #if NET8_0_OR_GREATER
+
+            Amazon.Lambda.Core.SnapshotRestore.RegisterBeforeSnapshot(async () =>
+            {
+                var beforeSnapstartRequests = GetBeforeSnapshotRequests();
+
+                foreach (var httpRequest in beforeSnapstartRequests)
+                {
+                    var invokeTimes = 5;
```

Comment:
Still wonder if this should be configurable somehow. Seems odd to just choose 5.

Reply:
We could expand this in a future version if need be.  Alternatively, if someone wanted to invoke an additional 5 more times, they could call `AddAWSLambdaBeforeSnapshotRequest` a second time.

At this iteration - I think exposing it would confuse users more, as they wouldn't know how to tune it effectively.  Or they'd feel they'd need to spend time figuring out an appropriate value.

Reply:
Ok, it will be interesting to see what feedback we get for this but this is a two way door.